starter-code/scripts/models/article.js

```diff
@@ -46,12 +46,22 @@ Article.fetchAll = function() {
     /* When our data is already in localStorage:
     1. We can process and load it,
     2. Then we can render the index page.  */
+    var blogArts = JSON.parse(localStorage.getItem('blogArticles'));
+    Article.loadAll(blogArts);
+    console.log('data already in localStorage');
```

Comment:
Good use of console.log to test logical flow! Once you confirm uses like these, it is typically best to remove them before pushing up.